src/user.js

```diff
@@ -1,29 +1,35 @@
 class User {
-  constructor(user) {
-    this.id = user.id;
-    this.name = user.name;
-    this.pantry = user.pantry;
-    this.favoriteRecipes = [];
-    this.recipesToCook = [];
-  }
-  saveRecipe(recipe) {
-    this.favoriteRecipes.push(recipe);
-  }
+	constructor(user) {
+		this.id = user.id;
+		this.name = user.name;
+		this.pantry = user.pantry;
+		this.favoriteRecipes = [];
+		this.recipesToCook = [];
+	}
 
-  removeRecipe(recipe) {
-    let i = this.favoriteRecipes.indexOf(recipe);
-    this.favoriteRecipes.splice(i, 1);
-  }
+	favoriteRecipe(recipe) {
+		this.favoriteRecipes.push(recipe);
+		// return this.favoriteRecipes
+	}
 
-  decideToCook(recipe) {
-    this.recipesToCook.push(recipe);
-  }
-  filterRecipes(type) {
-    return this.favoriteRecipes.filter(recipe => recipe.type.includes(type));
-  }
-  searchForRecipe(keyword) {
-    return this.favoriteRecipes.filter(recipe => recipe.name.includes(keyword) || recipe.ingredients.includes(keyword));
-  }
+	removeFavoriteRecipe(recipe) {
+		let i = this.favoriteRecipes.indexOf(recipe);
```

Comment:
I agree with your thought on these; from what I understand of the functionality, it makes sense to return something from these methods

Reply:
ok, great!  i'll comment those in then